src/creature/creature.cpp

```diff
@@ -5,11 +5,11 @@
 
 Creature::Creature()
 	: m_hp(10), m_ac(10), m_speed(30), m_ability_scores{10, 10, 10, 10, 10, 10},
-	  m_creature_type("humanoid"){};
+	  m_creature_type("humanoid"), m_display_name("main_character"){};
```

Comment:
It doesn't make sense for the `Creature` to initialize its display name to "main_character". Better initialize it to "creature", and then have the `MainCharacter` class initialize its own display name to "main character". The same for the other occurrences in this file.